[build] Can't build raylib on macOS

Hello,

I'm trying to build raylib on my mac for 10 hours:( but I get again and again the same error, as soon as I enter 

`make PLATFORM=PLATFORM_DESKTOP`

I get the following error. I realy hope someone kann help me:( i need raylib for a Project in the university 

> clang -c rglfw.c -O1 -Wall -std=c99 -D_DEFAULT_SOURCE -Wno-missing-braces -Werror=pointer-arith -fno-strict-aliasing -Werror=implicit-function-declaration -x objective-c -I. -Iexternal/glfw/include -Iexternal/glfw/deps/mingw -DPLATFORM_DESKTOP -DGRAPHICS_API_OPENGL_33
In file included from rglfw.c:54:
In file included from ./external/glfw/src/context.c:28:
In file included from ./external/glfw/src/internal.h:183:
In file included from ./external/glfw/src/cocoa_platform.h:30:
In file included from /System/Library/Frameworks/Carbon.framework/Headers/Carbon.h:24:
In file included from /System/Library/Frameworks/ApplicationServices.framework/Headers/ApplicationServices.h:27:
In file included from /System/Library/Frameworks/ApplicationServices.framework/Frameworks/ATS.framework/Headers/ATS.h:28:
In file included from /System/Library/Frameworks/ApplicationServices.framework/Frameworks/ATS.framework/Headers/ATSLayoutTypes.h:28:
In file included from /System/Library/Frameworks/ApplicationServices.framework/Frameworks/ATS.framework/Headers/ATSTypes.h:24:
In file included from /System/Library/Frameworks/CoreGraphics.framework/Headers/CGGeometry.h:8:
/System/Library/Frameworks/CoreGraphics.framework/Headers/CGBase.h:282:78: error: 
      'swift_name' attribute cannot be applied to this declaration
  ...CF_BRIDGED_TYPE(id) __IOSurface *IOSurfaceRef __attribute__((swift_name(...
                                                                  ^
In file included from rglfw.c:111:
./external/glfw/src/cocoa_init.m:77:5: error: use of undeclared identifier
      'NSString'; did you mean 'cString'?
    NSString* appName = nil;
    ^~~~~~~~
    cString
/System/Library/Frameworks/Carbon.framework/Frameworks/OpenScripting.framework/Headers/ASRegistry.h:212:3: note: 
      'cString' declared here
  cString                       = cStringClass /* old name for cStringCl...
  ^
In file included from rglfw.c:111:
./external/glfw/src/cocoa_init.m:77:15: error: use of undeclared identifier
      'appName'; did you mean 'pName'?
    NSString* appName = nil;
              ^~~~~~~
              pName
/System/Library/Frameworks/CoreServices.framework/Frameworks/AE.framework/Headers/AERegistry.h:478:3: note: 
      'pName' declared here
  pName                         = 'pnam' /*  0x706e616d  */
  ^
In file included from rglfw.c:111:
./external/glfw/src/cocoa_init.m:77:23: error: expression is not assignable
    NSString* appName = nil;
    ~~~~~~~~~~~~~~~~~ ^
./external/glfw/src/cocoa_init.m:78:5: error: use of undeclared identifier
      'NSDictionary'; did you mean 'UseDictionary'?
    NSDictionary* bundleInfo = [[NSBundle mainBundle] infoDictionary];
    ^~~~~~~~~~~~
    UseDictionary
/System/Library/Frameworks/ApplicationServices.framework/Frameworks/SpeechSynthesis.framework/Headers/SpeechSynthesis.h:1504:1: note: 
      'UseDictionary' declared here
UseDictionary(
^
In file included from rglfw.c:111:
./external/glfw/src/cocoa_init.m:78:5: warning: 'UseDictionary' is deprecated:
      first deprecated in OS X 10.8 [-Wdeprecated-declarations]
    NSDictionary* bundleInfo = [[NSBundle mainBundle] infoDictionary];
    ^
/System/Library/Frameworks/ApplicationServices.framework/Frameworks/SpeechSynthesis.framework/Headers/SpeechSynthesis.h:1504:1: note: 
      'UseDictionary' has been explicitly marked deprecated here
UseDictionary(
^
In file included from rglfw.c:111:
./external/glfw/src/cocoa_init.m:78:19: error: use of undeclared identifier
      'bundleInfo'
    NSDictionary* bundleInfo = [[NSBundle mainBundle] infoDictionary];
                  ^
./external/glfw/src/cocoa_init.m:78:32: error: expected expression
    NSDictionary* bundleInfo = [[NSBundle mainBundle] infoDictionary];
                               ^
./external/glfw/src/cocoa_init.m:79:5: error: use of undeclared identifier
      'NSString'; did you mean 'cString'?
    NSString* nameKeys[] =
    ^~~~~~~~
    cString
/System/Library/Frameworks/Carbon.framework/Frameworks/OpenScripting.framework/Headers/ASRegistry.h:212:3: note: 
      'cString' declared here
  cString                       = cStringClass /* old name for cStringCl...
  ^
In file included from rglfw.c:111:
./external/glfw/src/cocoa_init.m:79:24: error: expected expression
    NSString* nameKeys[] =
                       ^
./external/glfw/src/cocoa_init.m:79:15: error: use of undeclared identifier
      'nameKeys'
    NSString* nameKeys[] =
              ^
./external/glfw/src/cocoa_init.m:80:5: error: expected expression
    {
    ^
./external/glfw/src/cocoa_init.m:88:29: error: use of undeclared identifier
      'nameKeys'
    for (i = 0;  i < sizeof(nameKeys) / sizeof(nameKeys[0]);  i++)
                            ^
./external/glfw/src/cocoa_init.m:88:48: error: use of undeclared identifier
      'nameKeys'
    for (i = 0;  i < sizeof(nameKeys) / sizeof(nameKeys[0]);  i++)
                                               ^
./external/glfw/src/cocoa_init.m:90:19: error: use of undeclared identifier
      'bundleInfo'
        id name = bundleInfo[nameKeys[i]];
                  ^
./external/glfw/src/cocoa_init.m:90:30: error: use of undeclared identifier
      'nameKeys'
        id name = bundleInfo[nameKeys[i]];
                             ^
./external/glfw/src/cocoa_init.m:92:13: error: expected expression
            [name isKindOfClass:[NSString class]] &&
            ^
./external/glfw/src/cocoa_init.m:95:13: error: use of undeclared identifier
      'appName'
            appName = name;
            ^
./external/glfw/src/cocoa_init.m:100:10: error: use of undeclared identifier
      'appName'; did you mean 'pName'?
    if (!appName)
         ^~~~~~~
         pName
/System/Library/Frameworks/CoreServices.framework/Frameworks/AE.framework/Headers/AERegistry.h:478:3: note: 
      'pName' declared here
  pName                         = 'pnam' /*  0x706e616d  */
  ^
In file included from rglfw.c:111:
./external/glfw/src/cocoa_init.m:104:13: error: use of undeclared identifier
      'appName'
            appName = @(*progname);
            ^
fatal error: too many errors emitted, stopping now [-ferror-limit=]
1 warning and 20 errors generated.
make: *** [rglfw.o] Error 1

--- a/src/Makefile
+++ b/src/Makefile
@@ -221,7 +221,7 @@ ifeq ($(PLATFORM),PLATFORM_DESKTOP)
     ifeq ($(PLATFORM_OS),OSX)
         # OSX default compiler
         CC = clang
-        GLFW_CFLAGS = -x objective-c
+        GLFW_CFLAGS = -ObjC
     endif
     ifeq ($(PLATFORM_OS),BSD)
         # FreeBSD, OpenBSD, NetBSD, DragonFly default compiler

